### ISSUE TITLE
Exchange Listings

### ISSUE THREAD

**Issue by JSterling8** [turtlecoin/meta#137]
Are we interested in listing on Kraken and Liquid? I feel like we'd all like everything to be done as transparently as possible, and if an exchange asks for a listing fee I don't think we should list with them.

I have contacted Kraken and they've sent a listing application form. Assuming there's interest, who should I forward it onto? I could fill it in myself, but it asks a lot of Q's I don't know the answer to (most I can just leave blank though). Happy to do my best effort filling it in, but I think somebody who's established in the community should. We can also abandon it entirely if there's no consensus on applying at Kraken.

I also have a connection at Liquid that has offered to loop in their Biz Dev team with the TurtleCoin team. How should we proceed?

**Comment by SoreGums** [comment]
In general there is not going to be that much  input from the developers. This project is about the tech, hence the ban on market talk in the discord server.

So with that in mind and "it's a doers market" it's up to people to do things.

If an exchange wants to list TurtleCoin then they do the work to list TurtleCoin, if TurtleCoin puts out an update, then the Exchange does what is needed if they want TurtleCoin. There is no bizdev over here in the TurtleCoin project nor much interest in wasting time talking to them.

At the same time anyone that wants to contribute to TurtleCoin the project is welcome.  That simply looks different to "pay me some x value and I'll do y".

Hopefully that gives some insights @JSterling8 :)

**Comment by JSterling8** [comment]
@SoreGums - Thanks for your insight. It lines up with the general feeling I got from lurking the Discord and talking to some community members. I will probably just forge ahead with this and see what comes of it. I think it's probably best both ethically and legally for devs to not pay it any attention. They need to just be able to stick to the (completely true) line of: "We just made a blockchain for fun. Other people got it listed on exchanges and started trading it, but we made no efforts to do so."
